app/src/main/AndroidManifest.xml

```diff
@@ -24,6 +25,11 @@
         android:supportsRtl="true"
         android:theme="@style/AppThemeWhite"
         tools:ignore="GoogleAppIndexingWarning,RtlEnabled">
+        <meta-data
+            android:name="com.google.android.apps.drive.
+            387849404801-r4notopkkhuith55c84273mr3a9k3e5a.apps.googleusercontent.com"
```

Comment:
@Swati4star After merging it on your side, please make sure to make a new project in Google API Console and get an APP ID from there and replace it here. If deployed otherwise, the bill will come due to my account.